models/Element_OphTrOperationnote_Biometry.php

```diff
@@ -0,0 +1,133 @@
+ * You should have received a copy of the GNU General Public License along with OpenEyes in a file titled COPYING. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package OpenEyes
+ * @link http://www.openeyes.org.uk
+ * @author OpenEyes <info@openeyes.org.uk>
+ * @copyright Copyright (c) 2008-2011, Moorfields Eye Hospital NHS Foundation Trust
+ * @copyright Copyright (c) 2011-2013, OpenEyes Foundation
+ * @license http://www.gnu.org/licenses/gpl-3.0.html The GNU General Public License V3.0
+ */
+
+/**
+ * This is the model class for table "element_procedurelist".
+ *
+ * The followings are the available columns in table 'element_operation':
+ * @property string $id
+ * @property integer $event_id
+ * @property integer $surgeon_id
+ * @property integer $assistant_id
+ * @property integer $anaesthetic_type
+ *
+ * The followings are the available model relations:
+ * @property Event $event
+ */
+class Element_OphTrOperationnote_Biometry extends Element_OpNote
+{
+	// these are legacy and should be removed one switch to using the constants on the Eye model
+	const LEFT = Eye::LEFT;
+	const RIGHT = Eye::RIGHT;
+	const BOTH = Eye::BOTH;
+
+	private $event;
+
+	/**
+	 * @return bool
+	 */
+	public function hasLeft()
+	{
+		return $this->eye_id != Eye::RIGHT;
+	}
+
+	/**
+	 * @return bool
+	 */
+	public function hasRight()
+	{
+		return $this->eye_id != Eye::LEFT;
+	}
+
+	/**
+	 * Returns the static model of the specified AR class.
+	 * @return ElementOperation the static model class
+	 */
+	public static function model($className = __CLASS__)
+	{
+		return parent::model($className);
+	}
+
+	/**
+	 * @return string the associated database table name
+	 */
+	public function tableName()
+	{
+		return 'et_ophtroperationnote_biometry';
+	}
+
+	/**
+	 * @return array customized attribute labels (name=>label)
+	 */
+	public function attributeLabels()
+	{
+		return array(
+			'id' => 'ID',
+			'lens_right' => 'Lens',
+			'lens_left' => 'Lens',
+			'lens_id_right' => 'Lens',
+			'lens_id_left' => 'Lens',
+			'lens_description_left' => 'Description',
+			'lens_description_right' => 'Description',
+			'lens_acon_left' => 'A constant',
+			'lens_acon_right' => 'A constant',
+			'k1_left' => 'K1 (D)',
+			'k1_right' => 'K1 (D)',
+			'k2_left' => 'K2 (D)',
+			'k2_right' => 'K2 (D)',
+			'axis_k1_left' => 'Axis K1 (D)',
+			'axis_k1_right' => 'Axis K1 (D)',
+			'axial_length_left' => 'Axial length (mm)',
+			'axial_length_right' => 'Axial length (mm)',
+			'snr_left' => 'SNR',
+			'snr_right' => 'SNR',
+			'iol_power_left' => 'IOL Power',
+			'predicted_refraction_left' => 'Predicted Refraction',
+			'iol_power_right' => 'IOL Power',
+			'predicted_refraction_right' => 'Predicted Refraction',
+			'target_refraction_left' => 'Target Refraction',
+			'target_refraction_right' => 'Target Refraction',
+		);
+	}
+
+	public function findAll($attributes, $values){
```

Comment:
This declaration doesn't match the parent.
Declaration of Element_OphTrOperationnote_Biometry::findAll() should be compatible with that of CActiveRecord::findAll()